src/clj/rems/api/schema.clj

```diff
@@ -189,9 +196,8 @@
                                              :application/external-id s/Str}]
    :application/last-activity DateTime
    :application/applicant s/Str
-   :application/applicant-attributes {s/Keyword s/Str}
-   :application/members #{{:userid s/Str
-                           s/Keyword s/Str}}
+   :application/applicant-attributes UserWithAttributes
```

Comment:
We could replace these two with just `:application/applicant UserWithAttributes`